resume.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+  <html>
+    <head>
+      <title> Rossy </title>
+        	<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400&display=swap" rel="stylesheet">
+    		<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+      		<style>
+                  body {
+    padding-left: 10%;
+    padding-right: 10%;
+    margin-bottom: 10%;
+    background-color: #c3fbf4;
+}
+.Sticky {
+    width: 20vw;
+    height: 30vh;
+    position: fixed;
+    top: 0px;
+    z-index: 0;
+
+}
+.Main {
+    margin-top: 45vh;
+    background-color: #FF6978;
+    width: 80vw;
+    height: relative;
+    position: relative;
+    bottom: 30%;
+    z-index: 1;
+    opacity: 80%;
+}
+text {
+    font-family: "Lato", sans-serif;
+}
+h1 {
+    font-family: "Lato";
+	font-size: 3.5em;
+	text-indent: 0px;
+}
+h2 {
+    font-family: "Lato";
+	font-size: 2.2em;
+    text-indent: 0px;
+}
+h3 {
+    font-family: "Lato";
+    font-size: 1.3em;
+    text-indent: 0px;
+}
+p {
+    font-family: "Lato";
+    font-size: 1em;
+}
+.panel h3 {
+  color: #395C6B;
+}
+.panel p {
+  color: #395C6B;
+}
+.accordion {
+  background-color: #E3BAC6;
+  color: #596475;
+  cursor: pointer;
+  padding: 18px;
+  width: 100%;
+  text-align: left;
+  border: none;
+  outline: none;
+  transition: 0.5s;
+}
+.active, .accordion:hover {
+  background-color: #395C6B;
+  color: white;
+}
+
+.panel {
+  padding-left: 5vw;
+  padding-right: 5vw;
+  background-color: #B0D0D3;
+  max-height: 0;
+  overflow: hidden;
+  transition: max-height 0.2s ease-out;
+}
+</style>
+    </head>
+
+    <body> 
+        <div class="TopBar">
+        </div>
+        <div class="Sticky">
+          <h1> Rossy Yang </h1>
+          <h2> (631) 988-3820 </h2>
+          <h2> rossyyang@gmail.com </h2>
+        </div>
+        <div class="Main">
+         <button class="accordion"> Education </button>
```

Comment:
All `button` elements had a default `type` attribute of `submit` (which is the action the button takes when you submit a form); as a result, I recommend all `button` elements _always_ include a `type` attribute with a value to override this, like so:
```suggestion
         <button type="button" class="accordion"> Education </button>
```

The three valid attributes for a button `type` are:
- submit (the default)
- reset (as in, reset this form and clear out its fields)
- button (which is a more sensible default 🤷🏻‍♀️ and what you want here)

Reply:
Also, just because you're using a `button` element here doesn't mean you can't also use semantic headings. Consider instead:
```suggestion
         <h2><button class="accordion"> Education </button></h2>
```